howto/inference.md

```diff
@@ -15,7 +15,8 @@ The script assumes you have downloaded and converted the weights and saved them
 ```bash
 python generate.py --prompt "Hello, my name is" \
     --checkpoint_path ./checkpoints/lit-llama/7B/state_dict.pth \
-    --tokenizer_path ./checkpoints/lit-llama/tokenizer.model
+    --tokenizer_path ./checkpoints/lit-llama/tokenizer.model \
+    --model_size 7B
```

Comment:
I would avoid adding these, as #160 will drop the need for it. Or I miss something?

Reply:
These are added to clarify that the model size can be changed. And used only to access a specific path. I'm not sure if we want #160 to impose specific path structures, maybe yes.

But I propose adding it here and removing them in #160 if they become unnecessary